File: lua/refactoring/tests/refactor/106/go/struct-2/extract.start.go
package main

import "fmt"

type foobar struct {
   poggers int
}

func (b *foobar) simple_function(a int) {
    var test int = 1
    test_other := 1
    for idx := test - 1; idx < test_other; idx++ {
        fmt.Println(b.poggers, idx, a)
    }
}

File: lua/refactoring/tests/refactor/106/ts/class/extract.start.ts
class RefuctorPlease {
    multiple_returns(a: number) {
        let test: number = 1;
        let test_other = 11
        for (let idx = test - 1; idx < test_other; ++idx) {
            console.log(idx, a)
        }

        return {test, test_other};
    }
}
